src/main/kotlin/com/pubnub/api/managers/RetrofitManager.kt

```diff
@@ -25,13 +25,13 @@ import java.util.concurrent.TimeUnit
 
 class RetrofitManager(val pubnub: PubNub) {
 
-    private lateinit var transactionClientInstance: OkHttpClient
+    private lateinit var transactionClientInstance: OkHttpClient // TODO it doesn't seem right that these are lateinit
 
     private lateinit var subscriptionClientInstance: OkHttpClient
 
     private lateinit var noSignatureClientInstance: OkHttpClient
 
-    private var signatureInterceptor: SignatureInterceptor
+    private val signatureInterceptor: SignatureInterceptor
 
     internal val timeService: TimeService
```

Comment:
Why this is needed?

Reply:
We'll do cleanups like this in a separate PR, removing for now.